src/Npgsql/NpgsqlTransaction.cs

```diff
@@ -30,7 +30,7 @@ public sealed class NpgsqlTransaction : DbTransaction
 
         // Note that with ambient transactions, it's possible for a transaction to be pending after its connection
         // is already closed. So we capture the connector and perform everything directly on it.
-        readonly NpgsqlConnector _connector;
+        NpgsqlConnector _connector;
```

Comment:
Is this intentional?

Reply:
Nope, just something from when I was testing.